### PR TITLE
feat: adds Text Input UI component

### PR DESCRIPTION
**Pull request by Woolfs1423**
## Description

Inputs are used on the `form` page. Elements are created to get the last name, first name and middle name of the user.

## Task Link

[Link to the Figma design](https://www.figma.com/file/quYFY10igqGb8z8IUT6dp0/HTML-2-_-%D0%9F%D0%B8%D0%BD%D0%BA?type=design&node-id=1226%3A599&mode=design&t=3dyUYJJqK9WvL1ZH-1)

## Usage 

When you add a new `text-input` you need to change the `placeholder` and `id` for the inputs. Also change the label. The `for` in the label and the `id` in the input must be the same, they are connected.

## Checklist

Before submitting this pull request, please ensure the following:

- [x] I've validated my HTML with [Nu Html Checker](https://validator.w3.org/nu/), and it outputs no errors
- [x] I've checked my HTML with [HTML Tree Generator](https://yoksel.github.io/html-tree/en/), and it outputs no errors
- I've ensured that the layout matches the mockup for the following control points:
  - [x] 1200px
  - [x] 660px
  - [x] 320px

### CODE REVIEW

**Comment by naumch1k** [comment]
@MarPostovik @Woolfs1423 Please rebase as described [here](https://github.com/p-foundation/pink-app/pull/39#pullrequestreview-1987405665) and utilize the newly created `--common-transition` variable for your component

**Comment by Woolfs1423** [comment]
@naumch1k  I pulled down the new commits and replaced the transformation, with a variable

File: pages/components.css [hunk 1]
```diff
@@ -58,5 +58,10 @@
 @import url(../blocks/input/__wrapper/_type/input__wrapper_type_phone.css);
 @import url(../blocks/input/__wrapper/_type/input__wrapper_type_email.css);
 
+/* text-input UI Component*/
```
Comment (naumch1k) [inline pages/components.css:61]: @MarPostovik This comment' formatting a bit off comparing to other comments

**Review by naumch1k — CHANGES_REQUESTED** [review]
Hey, @MarPostovik! I left a few comments. Would you please go through these changes one last time to check if everything is okay with you?

Since we just merged #29 there are now conflicts in files `components.html` and `components.css`. Would you guys please take care of them so PR is ready to merge? 

Thank you ❤️

File: blocks/text-input/__input/text-input__input.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    border: 2px solid rgb(210, 40, 86);
+}
+.text-input__input::placeholder {
+    font-size: 18px;
+    color: rgb(40, 54, 69);
+    font-family: OpenSans, arial, sans-serif;
+    line-height: 30px;
+    letter-spacing: 0.5px;
+}
+@media screen and (max-width: 659px) {
```
[the remark this answers is not in the document]
Comment (MarPostovik) [reply blocks/text-input/__input/text-input__input.css]: Fixed ✅

File: pages/components.css (hunk 1 above)
Comment (MarPostovik) [reply pages/components.css]: Changed ✅

**Comment by MarPostovik** [comment]
Hi @naumch1k . I made changes and resolved conflicts.

**Review by naumch1k — APPROVED** [review]

**Pull request merged**